src/view/reviewManager.ts

```diff
@@ -729,6 +729,48 @@ export class ReviewManager {
 		return method;
 	}
 
+	private async _chooseTargetBranch(targetRemote: RemoteQuickPickItem, pullRequestDefaults: PullRequestDefaults): Promise<string | undefined> {
+		const base: string = targetRemote.remote
+			? (await this._folderRepoManager.getMetadata(targetRemote.remote.remoteName)).default_branch
+			: pullRequestDefaults.base;
+
+		let target: string | undefined;
+		try {
+			const branches = await this._folderRepoManager.listBranches(targetRemote.owner, targetRemote.name);
+			target = await new Promise(async (resolve, reject) => {
+				const branchPicker = vscode.window.createQuickPick();
```

Comment:
Can you use the simpler `showQuickPick` here instead?

Reply:
I don't think so, I don't see a way to set an initial value with the options for it